packages/lit-dev-content/site/articles/article/article.11tydata.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   eleventyComputed: {
-    layout: "article",
-    permalink: data => `/articles/${data.page.fileSlug}/`,
-  }
-}
\ No newline at end of file
+    layout: 'article',
+    permalink: (data) => `/articles/${data.page.fileSlug}/`,
+  },
+  tags: ['articles'],
```

Comment:
Is this for search index?

Reply:
if we tag each article we can just use the 11ty data system's `data.collections.articles` to get all the frontmatter data rather than a bunch of `fs` crawls

Reply:
see https://github.com/lit/lit.dev/pull/1265/files#diff-be43ec0405c08762a256d8e94941b4f09d56155982efefd012c7bdd3021d2409R54

Reply:
That is nice!